indico/modules/groups/__init__.py

```diff
@@ -21,7 +22,7 @@
 
 @signals.menu.items.connect_via('admin-sidemenu')
 def _extend_admin_menu(sender, **kwargs):
-    if session.user.is_admin:
+    if session.user.is_admin and config.LOCAL_GROUPS:
```

Comment:
For the reason mentioned in my other comment, this change needs to be reverted as well since there can still be remote groups. :/